src/moin/converter/markdown_in.py

```diff
@@ -58,10 +58,12 @@ def postproc_text(markdown, text):
     if text is None:
         return None
 
-    for pp in markdown.postprocessors.values():
+    for pp in markdown.postprocessors:
         text = pp.run(text)
 
     if text.startswith('<pre>') or text.startswith('<div class="codehilite"><pre>'):
+        # with markdown 3.0.0 indented code is somehow escaped twice
+        text = text.replace('&amp;lt;', '&lt;').replace('&amp;gt;', '&gt;').replace('&amp;amp;', '&amp;')
```

Comment:
`werkzeug.unescape(text)` maybe?

Reply:
I was using that at first, but then worried it may do too much. I wanted to reverse just these 3 strings from markdown serializers.py:

    def _escape_cdata(text):
        # escape character data
        try:
            # it's worth avoiding do-nothing calls for strings that are
            # shorter than 500 character, or so.  assume that's, by far,
            # the most common case in most applications.
            if "&" in text:
                # Only replace & when not part of an entity
                text = RE_AMP.sub('&amp;', text)
            if "<" in text:
                text = text.replace("<", "&lt;")
            if ">" in text:
                text = text.replace(">", "&gt;")
            return text
        except (TypeError, AttributeError):  # pragma: no cover
            _raise_serialization_error(text)

If you prefer the workzeug, I will change

Reply:
huh, strange...